tests/functional/test_api.py

```diff
@@ -88,6 +100,30 @@ def user_with_token(user, db_session, factories):
 
 
 @pytest.fixture
+def auth_client(db_session, factories):
+    auth_client = factories.AuthClient(authority='thirdparty.example.org')
+    db_session.add(auth_client)
```

Comment:
FWIW you [don't need to explicitly add](https://github.com/hypothesis/h/blob/b8024a0fc0ea37bd2fdc54d2d3adba087aff116d/tests/common/factories.py#L50) ORM objects created through the `factories` fixture.